### ISSUE TITLE
Failed to create FASTQ output

### ISSUE THREAD

**Issue by PavelSenkyrik** [epi2me-labs/wf-basecalling#31]
### Operating System

Ubuntu 22.04

### Workflow Version

b1.1.5.

### Workflow Execution

EPI2ME Desktop application

### EPI2ME Version

v5.1.9.

### Workflow Execution - CLI Execution Profile

standard (default)

### What happened?

Hello I successfully ran this workflow to duplex and map my data. But I was wondering, why from simplex 16,000 reads we have only 1,500 reads after the barcoding (We used MinKNOW for barcoding). So I wanted to first re-basecall our data to get a higher read quality and set the output to the FASTQ file so I can barcode the data again in the MinKNOW. However, this time I get this error. Is there something wrong with my settings or I just need to use different tools. 

### Relevant log output

```shell
N E X T F L O W  ~  version 23.04.2
Launching `/home/nanopore/epi2melabs/workflows/epi2me-labs/wf-basecalling/main.nf` [HGC_1_Unclassified] DSL2 - revision: 6d842f59e6
WARN: NEXTFLOW RECURSION IS A PREVIEW FEATURE - SYNTAX AND FUNCTIONALITY CAN CHANGE IN FUTURE RELEASES
||||||||||   _____ ____ ___ ____  __  __ _____      _       _
||||||||||  | ____|  _ \_ _|___ \|  \/  | ____|    | | __ _| |__  ___
|||||       |  _| | |_) | |  __) | |\/| |  _| _____| |/ _` | '_ \/ __|
|||||       | |___|  __/| | / __/| |  | | |__|_____| | (_| | |_) \__ \
||||||||||  |_____|_|  |___|_____|_|  |_|_____|    |_|\__,_|_.__/|___/
||||||||||  wf-basecalling v1.1.5
--------------------------------------------------------------------------------
Core Nextflow options
  runName        : HGC_1_Unclassified
  containerEngine: docker
  launchDir      : /home/nanopore/epi2melabs/instances/wf-basecalling_01HPPE2SN2JSYMWN17PN90KNHR
  workDir        : /home/nanopore/epi2melabs/instances/wf-basecalling_01HPPE2SN2JSYMWN17PN90KNHR/work
  projectDir     : /home/nanopore/epi2melabs/workflows/epi2me-labs/wf-basecalling
  userName       : nanopore
  profile        : standard
  configFiles    : /home/nanopore/epi2melabs/workflows/epi2me-labs/wf-basecalling/nextflow.config
Input Options
  input          : /home/nanopore/Desktop/Nanopore/Zavadeci_Experimenty_zde/07_02_2024_HGC_1_Amplikony_multiplex/07_02_2024_HGC_1/20240207_1136_MN41306_FAY35404_d45e9a0c/pod5_pass/unclassified
Output Options
  out_dir        : /home/nanopore/epi2melabs/instances/wf-basecalling_01HPPE2SN2JSYMWN17PN90KNHR/output
  fastq_only     : true
Basecalling options
  basecaller_cfg : dna_r10.4.1_e8.2_400bps_sup@v4.3.0
  duplex         : true
!! Only displaying parameters that differ from the pipeline defaults !!
--------------------------------------------------------------------------------
If you use epi2me-labs/wf-basecalling for your analysis please cite:
* The nf-core framework
  https://doi.org/10.1038/s41587-020-0439-x
--------------------------------------------------------------------------------
This is epi2me-labs/wf-basecalling v1.1.5.
--------------------------------------------------------------------------------
WARN: Reads are not filtered when using bonito.
WARN: Detected input files with an unknown naming pattern. These will be basecalled, but duplex rates may be impacted.
[fd/b1e53b] Submitted process > getVersions
[56/a84df2] Submitted process > getParams
[12/58bebc] Submitted process > wf_dorado:dorado (1)
[d8/019542] Submitted process > wf_dorado:dorado_summary (1)
[92/4bb8ed] Submitted process > wf_dorado:qsFilter (1)
[28/ba0898] Submitted process > wf_dorado:combine_dorado_summaries
[22/831047] Submitted process > pair_stats (1)
[da/690789] Submitted process > bamstats (1)
[ac/70cd63] Submitted process > wf_dorado:merge_fail_calls_to_fastq
[0d/113a2b] Submitted process > wf_dorado:merge_pass_calls_to_fastq
[d0/8e601c] Submitted process > progressive_pairings (1)
[16/0764ad] Submitted process > progressive_stats (1)
WARN: Input tuple does not match input set cardinality declared by process `split_xam` -- offending value: /home/nanopore/epi2melabs/instances/wf-basecalling_01HPPE2SN2JSYMWN17PN90KNHR/work/0d/113a2bcff5cfe39619bb808a99ce86/SAMPLE.pass.fq.gz
ERROR ~ Error executing process > 'split_xam (1)'
Caused by:
  Not a valid path value type: org.codehaus.groovy.runtime.NullObject (null)
Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh`
 -- Check '/home/nanopore/epi2melabs/instances/wf-basecalling_01HPPE2SN2JSYMWN17PN90KNHR/nextflow.log' file for details
WARN: Killing running tasks (1)
```

### Were you able to successfully run the latest version of the workflow with the demo data?

yes

**Comment by RenzoTale88** [comment]
Hi @PavelSenkyrik the `--duplex` option require the output of dorado to be in BAM format, as this contains the tags required to separate the duplex and simplex reads. We are working on a patch to make the options mutually exclusive, we apologise for the inconvenience.

**Comment by RenzoTale88** [comment]
@PavelSenkyrik this should be fixed as of version [v1.1.7](https://github.com/epi2me-labs/wf-basecalling/releases/v1.1.7)
